src/modules/coinmarket/CoinGeckoMarket.ts

```diff
@@ -26,9 +26,9 @@ export interface CoinMarket {
     fetch(): Promise<any>;
 }
 
-export class CoinGeckoMaket implements CoinMarket {
+export class CoinGeckoMarket implements CoinMarket {
     /**
-     * Market Client i.e Gecko_coin_market 
+     * Market Client i.e Gecko_coin_market
```

Comment:
are the underscores necessary?

Reply:
Editor trolling removal function. 
Service :)